benchmark/src/bench.rs

```diff
@@ -56,12 +57,21 @@ impl BenchmarkService for Benchmark {
     fn streaming_from_client(
         &mut self,
         ctx: RpcContext,
-        _: RequestStream<SimpleRequest>,
+        mut stream: RequestStream<SimpleRequest>,
         sink: ClientStreamingSink<SimpleResponse>,
     ) {
-        let f = sink.fail(RpcStatus::new(RpcStatusCode::UNIMPLEMENTED, None));
+        let f = async move {
+            let mut resp: Option<SimpleResponse> = None;
+            while let Some(req) = stream.try_next().await? {
```

Comment:
Can you reference the original implement?

Reply:
grpc/test/cpp/qps/server_sync.cc: 120

```c++
  static Status ClientPull(ServerContext* /*context*/, R* stream,
                           SimpleResponse* response) {
    SimpleRequest request;
    while (stream->Read(&request)) {
    }
    if (request.response_size() > 0) {
      if (!Server::SetPayload(request.response_type(), request.response_size(),
                              response->mutable_payload())) {
        return Status(grpc::StatusCode::INTERNAL, "Error creating payload.");
      }
    }
    return Status::OK;
  }
```

Reply:
Then the implement is wrong here, cpp constructs the response using the last request.

Reply:
I'll fix it